.aicoe-ci.yaml

```diff
@@ -2,12 +2,12 @@
 # Example `.aicoe-ci.yaml` with a full list of config options is available here: https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml
 check:
   # Uncomment following line to build a public image of this repo
-   - thoth-build
+  - thoth-build
 # Uncomment following lines to build a public image of this repo
 build:
    build-stratergy: Source
    base-image: quay.io/thoth-station/s2i-elyra-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+   registry: quay.io
+   registry-org: atersaak
+   registry-project: github-labeler
+   registry-secret: ewogICJhdXRocyI6IHsKICAgICJxdWF5LmlvIjogewogICAgICAiYXV0aCI6ICJZWFJsY25OaFlXc3JjbTlpYjNRNlRrNDRTRk5FU2xoSFZFRTRObGd6VXpRNFJEbEhNVU5LVGpVMFZrRkpORXBPT1RSWVZraFZWazFUTlRCT1ZFaEpRbFEyVVZFNVFrVTBOMFJHVmtzM1Z3PT0iLAogICAgICAiZW1haWwiOiAiIgogICAgfQogIH0KfQ
```

Comment:
you probably don't want to expose your secret like this :)

Reply:
I suggest you use aicoe org repo and secret name https://github.com/aicoe-aiops/ocp-ci-analysis/blob/88453659fcb988e813cdccdf007823ff30c0ad28/.aicoe-ci.yaml#L14

the secret should not be exposed anywhere, we need to improve on this https://github.com/AICoE/aicoe-ci/issues/90. Please talk to @harshad16 about this if you want your specific secret to be set.